plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -278,10 +282,15 @@ export class MySQLInstrumentation extends InstrumentationBase<
           values = [_valuesOrCallback];
         }
 
-        span.setAttribute(
-          SemanticAttributes.DB_STATEMENT,
-          getDbStatement(query, format, values)
-        );
+        const dbStatementSerializer =
+          typeof thisPlugin._config.dbStatementSerializer === 'function'
+            ? thisPlugin._config.dbStatementSerializer
+            : thisPlugin._defaultDbStatementSerializer.bind(thisPlugin);
+
+        let dbStatement = getDbStatement(query, format, values);
+        dbStatement = dbStatementSerializer(dbStatement);
```

Comment:
if you choose to keep `dbStatementSerializer` as a configuration function (and not boolean), then it should be wrapped in try-catch to protect instrumentation from any bad configuration that can throw